en/components/combo.md

```diff
@@ -106,6 +106,76 @@ When igxCombo is opened, allow custom values are enabled and add item button is
 
 - `ArrowUp` focus will be moved back to the last list item or if list is empty will be moved to the search input.
 
+## Styling
+Using the [Ignite UI for Angular Theming](../themes/index.md), we can greatly alter the **igx-combo** appearance. 
+Since `igx-combo` extends `igx-drop-down`, it also makes good use of its existing `igx-drop-down` styling, so you can directly refer to the [igx-drop-down styling guide](../themes/drop_down.md) for details.
+On top of that, `IgxCombo` includes an `IgxInputGroup` as well, so any styling to the input-group will affect the `IgxCombo` component. You can refer to [igx-input-group styling guide](../themes/input_group.md) for details.
+
+### Code snippets
+```scss
+// in component.scss
+@import '~igniteui-angular/lib/core/styles/themes/index';
+
+$my-primary-color: #ffffff;
+$my-secondary-color: #7344df;
+
+$my-color-palette: igx-palette(
+    $primary: $my-primary-color,
+    $secondary: $my-secondary-color
+);
+
+$my-light-purple: igx-color($my-color-palette, "secondary", 100);
+$my-medium-purple: igx-color($my-color-palette, "secondary", 300);
+$my-medium-plus-purple: igx-color($my-color-palette, "secondary", 400);
+$my-dark-purple: igx-color($my-color-palette, "secondary", 600);
+
+$custom-drop-down-theme: igx-drop-down-theme(
+    $palette: $my-color-palette,
+    $background-color: $my-light-purple,
+    $item-text-color: $my-primary-color,
+    $header-text-color: $my-dark-purple,
+
+    $selected-item-background: $my-medium-plus-purple,
+    $selected-item-text-color: $my-primary-color,
+    $selected-hover-item-background: $my-medium-plus-purple,
+    $selected-hover-item-text-color: $my-primary-color,
+    $selected-focus-item-background: $my-medium-plus-purple,
+    $selected-focus-item-text-color: $my-primary-color,
+
+    $focused-item-background: $my-medium-purple,
+    $focused-item-text-color: $my-primary-color,
+
+    $hover-item-background: $my-primary-color,
+    $hover-item-text-color: $my-dark-purple
+);
+
+// igx-combo-theme exposes several parameters on top of the igx-drop-down-theme.
+// change $search-separator-border-color to one matching better our purple theme
+$custom-combo-theme: igx-combo-theme(
+    $search-separator-border-color: $my-dark-purple
+);
+
+// Pass our custom-drop-down-theme and custom-combo-theme to respectively `igx-drop-down` and igx-combo mixins.
+:host {
```

Comment:
This should be a separate code snippet with a bit more explanation (and the note will be more easily associated with this part *exactly*)